client/mysql_plugin.c

```diff
@@ -385,6 +388,11 @@ static int get_default_values()
       {
         opt_plugin_ini= my_strdup(value, MYF(MY_FAE));
       }
+      if ((opt_lc_messages_dir == 0) && ((value= get_value(line, "--lc_messages_dir"))))
```

Comment:
this won't work if the option is specified as `lc-messages-dir`. Because `get_value()` need to compare `-` and `_` as equal, and it does not. Same bug with `get_value(line, "--plugin_dir")`

Also, there is no need to do `get_value(line, "--plugin_ini"))))` because server config can never have it, it's not a valid server option.

Reply:
Thx @vuvova I will fix it.